Assets/LeapMotion/Scripts/Utils/ValueMappingExtensions.cs

```diff
@@ -0,0 +1,125 @@
+﻿using UnityEngine;
+
+namespace Leap.Unity {
+
+  public static class ValueMappingExtensions {
```

Comment:
Well ValueMappingExtensions is the name of the class anyway, so no need for long using statements, its all in Leap.Unity :)